docs/advance-toolchain-documentation.html

```diff
@@ -347,11 +349,34 @@
 
 <p>YUM supports package upgrades for new revision releases (i.e. <i>5.0-0</i> to <i>5.0-1</i>). For new major releases (i.e. <i>5.0-8</i> to <i>6.0-1</i>), please proceed as in a normal installation.</p>
 <br>
+
+<h2><a name="powertools">Using IBM Linux on Power tools repository</a></h2>
+
+<p>Advance Toolchain is part of the IBM Linux on Power tools repository, you can get AT's repository configured by installing the ibm-power-repo package:</p>
+<ul>
+	<li>Download the RPM package from <a href="https://www14.software.ibm.com/webapp/set2/sas/f/lopdiags/yum.html">https://www14.software.ibm.com/webapp/set2/sas/f/lopdiags/yum.html</a>.</li>
+	<li>Install the ibm-power-repo RPM package, by running the command with root privileges:</li>
+<blockquote style="margin-left:40px">
+  <pre style="font-family:courier new,courier,monospace;">
+rpm -ivh ibm-power-repo-latest.noarch.rpm
+  </pre>
+</blockquote>
+	<li>After the installation, run the following command by using root user privileges:</li>
+<blockquote style="margin-left:40px">
+  <pre style="font-family:courier new,courier,monospace;">
+/opt/ibm/lop/configure
+  </pre>
+</blockquote>
+</ul>
+<p>Then you can install AT by using yum or zypper commands.</p>
+<br>
+
 <h2><a name="at_downloader">AT Downloader</a></h2>
 
 <p>There is a tool available to download AT called <a href="ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/at_downloader/">AT Downloader</a>.</p>
-<p>The  <a href="ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/at_downloader/">AT Downloader</a> is a script to download the latest version of the AT packages for a supported distribution.</p>
+<p>The  <a href="ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/at_downloader/">AT Downloader</a> is a script to download the latest version of the AT packages for a supported distribution and create an ISO image if desired.</p>
 <p>You can then manually install the AT packages on your systems.</p>
+<p><b>Note:</b> The prefered method to download AT is configuring the repository (Using <a href="#yast">Yast</a> or <a href="#yum">Yum</a>), the intention of this script is to help users that have limited internet access on their power servers.</p>
```

Comment:
Adding to tuliom's comments:
- s/Using/see/
- I'd mention all four sections about "configuring a repository": Yast, Yum, Aptitude, and IBM Power Repo